docs/core/proto-docs.md

```diff
@@ -1943,6 +1946,28 @@ GenesisState defines the nftmarket module's genesis state.
 
 
 
+<a name="ununifi.nftmarket.PaymentStatus"></a>
+
+### PaymentStatus
+
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| `nft_id` | [NftIdentifier](#ununifi.nftmarket.NftIdentifier) |  |  |
+| `bidder` | [string](#string) |  |  |
+| `amount` | [cosmos.base.v1beta1.Coin](#cosmos.base.v1beta1.Coin) |  |  |
+| `automatic_payment` | [bool](#bool) |  |  |
+| `paid_amount` | [string](#string) |  |  |
+| `bid_time` | [google.protobuf.Timestamp](#google.protobuf.Timestamp) |  |  |
+| `state` | [ListingState](#ununifi.nftmarket.ListingState) |  |  |
+| `all_paid` | [bool](#bool) |  |  |
```

Comment:
Can the table more formatted? There's automatic formatter extention on VScode

Reply:
Which extension of vscode do you use?